### ISSUE TITLE
read ATS file error

### ISSUE THREAD

**Issue by chdwql** [resistics/resistics#1]
#### Problem description
I try to read ATS files, but got error, test code and error as follows:
```python
from resistics.ioHandlers.dataReaderATS import DataReaderATS

# read in ATS data
atsReader = DataReaderATS(r'D:\Works\Python\cself\TS\256Hz')
atsReader.printInfo()
# now get some data
startTime = "2018-04-30 00:00:00"
stopTime = "2018-04-30 23:00:00"
# let's get some unscaled data
unscaledData = atsReader.getUnscaledData(startTime, stopTime)
unscaledData.printInfo()
# now let's look at the data
unscaledData.view(sampleEnd=20000)
# let's try physical data
physicalData = atsReader.getPhysicalData(startTime, stopTime)
physicalData.printInfo()
# let's view it
physicalData.view()
# but all we see is 50Hz and 16Hz noise
# so we can apply a low pass filter
physicalData.view(sampleEnd=20000, lpfilt=4)
```
#### error

![ERROR](https://user-images.githubusercontent.com/4538383/63565802-1424bc80-c59d-11e9-9676-743459970b08.png)

**Comment by nss350** [comment]
Hi,
Thanks for trying to use resistics and leaving a bug report. 

Can you provide some information about your setup:
Python version
What Metronix logger the data was recorded on
The files you have in the 256Hz directory

For some reason, the start_time header has not been found in the data. The easiest way for me to diagnose this issue is if you could share the data with me. What that be possible?

**Comment by chdwql** [comment]
> Hi,
> Thanks for trying to use resistics and leaving a bug report.
> 
> Can you provide some information about your setup:
> Python version
> What Metronix logger the data was recorded on
> The files you have in the 256Hz directory
> 
> For some reason, the start_time header has not been found in the data. The easiest way for me to diagnose this issue is if you could share the data with me. What that be possible?


How to share the xml header file with you?

**Comment by chdwql** [comment]
My python version is 3.7.2.
I install resistics by pip, version is 0.5.
![捕获](https://user-images.githubusercontent.com/4538383/63630254-4987e380-c64b-11e9-9017-2c08968a6614.PNG)

**Comment by nss350** [comment]
Hi again,
The python and resistics versions look fine. 

You can find contact information for resistics here:
https://resistics.io/contact.html

If you email the header file I can have a look. If it is possible for you to share all the data in the directory, that would be even better. Maybe through onedrive/dropbox etc. 

Thanks

**Comment by chdwql** [comment]
Please check your email. @nss350

**Comment by nss350** [comment]
Hi, I've now reproduced the error. The problem looks to be in the DataReaderATS class and with how it reads the xml file, particularly with relation to the measurement/recording/output section. 

Because I have no affiliation with Metronix, I don't have complete specification for their header files, so issues might occur until I have a better idea how they work.

I will work on this and let you know when a fix might be available.

Thanks

**Comment by chdwql** [comment]
Thanks~ @nss350

**Comment by nss350** [comment]
Commit 2d21112cb1203cd0c802fef99193c61786885eda is a fix released in version 0.0.6.dev1.

Closing the issue, but XML files are liable to break again if the formatting is changed.
